docs/android/release.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "Android Release Process"
+sidebar_label: "Release Process"
+---
```

Comment:
Didn't we talk about releasing in the ci file?

Reply:
Here it's more about the lifecycle of a release, but we can change that. Like you said the CI page is not very interesting for a lot of people.